src/Course.java

```diff
@@ -0,0 +1,11 @@
+public class Course {
+    private String name;
+
+    public String getName() {
+        return name;
+    }
+
+    public Course(String name) {
```

Comment:
constructors should be written after the class variables - so before the getter here